### DISCUSSION TITLE
No api refrence for wasmedge c++ SDK

### DISCUSSION THREAD

**Discussion by kumarutkarsh1248** [WasmEdge/WasmEdge#3537] [General]
I am trying to understand the plugin system in WasmEdge and I found that most of the plugins and example plugins are implemented in C++. However, I can't find any reference for the WasmEdge C++ SDK.

**Comment by q82419** [comment]
We don't have general C++ SDK for WasmEdge runtime yet.
For the plug-in, we have example docs of implementing WasmEdge plugin in C++.
https://wasmedge.org/docs/contribute/plugin/develop_plugin_cpp
